activemq-tooling/activemq-joram-jms-tests/src/main/java/org/objectweb/jtests/jms/conform/connection/TopicConnectionTest.java

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright 2009 Red Hat, Inc.
```

Comment:
I noticed there is a Red Hat reference in some files like here, I assume we need to get rid of this?

Reply:
Already corrected in the jakarta PR commit:

https://github.com/apache/activemq/pull/996/commits/17d7f024873765ba05ac735f53dc321117dc8881